lib/config.json

```diff
@@ -35,10 +35,14 @@
                 }
             }
         },
-        "hapi-pino":{
+        "hapi-pino": {
             "plugin": "require:hapi-pino",
             "options": {
-                "prettyPrint": true
+                "logEvents": {
+                    "$filter": "env.NODE_ENV",
+                    "$default": ["response", "request-error"],
+                    "development": false
+                }                 
             }
         }
```

Comment:
Based on our conversation... Drop the "logEvents" configuration and configure pino-pretty:
```
"hapi-pino": {
    "options": {
        "$filter": "env.NODE_ENV",
        "$default": {},
        "development": {
            "transport": {
                "target": "pino-pretty",
                "options": {
                    "colorize": true,
                    "translateTime": true
                }
            }
        }      
    }
}
```